drivers/usb/device/Kconfig

```diff
@@ -205,6 +205,32 @@ config USB_B91_WORK_QUEUE_STACK_SIZE
 	  for handling the events from the USBD ISR, i.e. executing endpoint
 	  callbacks and providing proper notifications to the USB device stack.
 
+config USB_TELINK_B92
```

Comment:
You can create common fields:
USB_TELINK_B9X
USB_B9X_EVT_QUEUE_SIZE
USB_B9X_WORK_QUEUE_STACK_SIZE

Reply:
Done